### ISSUE TITLE
Deposit anima / item upgrade blocked by addon

### ISSUE THREAD

**Issue by denisciumbargi** [GoldpawsStuff/AzeriteUI#126]
## The problem  

Unable to deposit anima or sometimes to use new item upgrade interface due to same error, action blocked by addon.

### Make sure you can check all these boxes  

- [ x] I have the latest addon version available from CurseForge.
- [ x] I restarted the entire game after updating the addon, not just relogged or reloaded. (Only needed in Classic.)

### What game version(s) does it happen in? Choose your version and the highest expansion you own. 

- Live Realms
	- [ ] Classic Era 
	- [ ] Burning Crusade Classic 
	- [ x] Shadowlands 
- Public Test Realm 
	- [ ] Classic Era 
	- [ ] Burning Crusade Classic 
	- [ ] Shadowlands 

### What addon theme(s) does it happen in?  

- [ x] Azerite 
- [ ] Legacy 
- [ ] I don't know what this means. 

### Does this happen with ONLY this addon enabled?  

- [x ] Yes 
- [ ] No 

### Does this happen with absolutely all addons (this included) disabled?  

- [ x] Yes 
- [ ] No 

### What classes does it happen on?  
ALL

### Any logs, screenshots, error output, etc?  

![image](https://user-images.githubusercontent.com/3941763/148524292-34ac74cf-0695-47ae-ae79-5267ab20510f.png)

### Steps to reproduce  

It's random. Doesn't always occur and not very often but when it does it's really hard to bypass unless u disable the addon temporarily. **Sometimes it happens in the new item upgrade interface also**.

### Expected behaviour  

Not to happen :-s

**Comment by goldpaw** [comment]
I've had this reported a few times over the course of this expansion. I fixed it the first time, and have been unable to reproduce the bug after that. It hasn't happened for me personally in actual years. 

I can't help thinking this occurs from a conflict with another addon. That somehow AzeriteUI and some other addon tries to affect some blizzard item at once, and the result - however random that might seem - is that the azerite deposit interface is tainted. 

Because AzeriteUI doesn't - intentionally - touch that part of the blizz UI or anything related to it. 

Can you give me a list of your addons, to give me a starting point here? Because I honestly have no idea where to begin. I need to be able to reproduce a bug in order to fix it.

**Comment by goldpaw** [comment]
Does a /relod before depositing fix it, btw? And by "fix" I just mean allows you to deposit. 

If not, then it's almost certainly an addon conflict.

**Comment by denisciumbargi** [comment]
Just tried /reload now and that indeed fixed it. Silly of me for not trying such an obvious solution.

Still, here are my addons, they are mostly yours although the list look big because some addons like to see their name for every little thing they load. Does that make a big fan of yours? ha ha :)

![image](https://user-images.githubusercontent.com/3941763/148526217-a259f06b-6173-482c-b7b5-4965b2162bc8.png)
![image](https://user-images.githubusercontent.com/3941763/148526269-df6bb8d8-9d35-4eef-a311-962b950d4392.png)
![image](https://user-images.githubusercontent.com/3941763/148526285-108d94e1-21a6-4621-9f73-b97f99573d99.png)

**Comment by goldpaw** [comment]
Well you have good taste in addons if nothing else, hehe! And always fun to see AzeriteUI in megaresolutions. I'm only on old 1920x1080 myself, so I've never actually experienced how it looks on proper highres. I mean, we made all the texture enormous for a reason! ;)

Anyway, that list didn't make me smarter, but thanks for it anyway. I can reference against this if I get other reports. 

The main problem is that this is what we refer to as a "taint", not a direct bug. A taint is when some addon changes something, and some execution path in the WoW UI that handles so called "secure" actions enocunter that change, and then decides some addon has manipulated code that shouldn't have been changed, and they then prevent the action from happening. 

This is part of Blizzard's "bot protection", but it sadly affects completely harmless addons, a lot. And the taint reports if we ever get them? They just name the addon that discovered the problem, not the addon that caused it. A taint report is really just naming the person that found the dead body, not the killer. Good analogy? :D

Good to hear /reload might help, but of course this is NOT a fix, just a clunky workaround.

**Comment by denisciumbargi** [comment]
Ha ha thanks and glad I could share the mega resolution with you :P You will get to that soon I am sure in the near future :P 

Believe it or not even still I find the screen cluttered, would love a feature like in combat to auto-hide quest panel to make it cleaner and focused on targets, who needs quests panel when engaged in combat
Maybe I'll put that as a request here if u think it's worth it :)))

About the taint and the issue, isn't happening often and is not a game-changer, just inconvenient, and yes, blizzard tends to do things messy for creators but hey, we all love a challenge at times, makes life less boring ha ha ha

**Comment by goldpaw** [comment]
What we have to remember, though, is that all the blizzard limitations comes from them trying to prevent botters from having major advantages over us non-cheaters. It sucks from an addon-development point of view, but in the end, there limitations are there for our sake. 

The tracker COULD be hidden in combat, yes. That is doable. No idea if it would cause taint or not, and it certainly will cause problems in quests where you have to click quest items, as it would require you to open the bags and locate the item. 

But as an option, it could be done. I could "sneak" in a row of options for the tracker, where "hide in combat" would be one of them. 

Can you make a new issue report with a feature request to hide the quest tracker in combat, please? I guarantee you I will forget otherwise. When I have my issue handling days, I read titles first and foremost. Do not trust my memory, but do trust my ability to read! :D

**Comment by denisciumbargi** [comment]
Will do that now. And thanks for the support, always prompt in replying :)

I understand the limitations and you have a point. 

And the part about trusting your memory.... believe me, I know the feeling :-s, not a good one ha ha, but happens to all of us intelligent people, we have too many great ideas to keep track of them all isn't it? :)))

**Comment by goldpaw** [comment]
Yeah my _replies_ are fast. Let's not talk about my actual fixing speed. Let's just keep talking about what's good. Yes.

**Comment by denisciumbargi** [comment]
![image](https://user-images.githubusercontent.com/3941763/148546387-b2aa23a3-1275-44e9-ae4e-f48f10d763a4.png)

Just happened now on the item upgrade :)

/Reload worked again.
